build.gradle.kts

```diff
@@ -62,164 +36,38 @@ dependencies {
     testImplementation("org.assertj:assertj-core:3.19.0")
-
-}
-
-// Set the JVM language level used to build the project. Use Java 11 for 2020.3+, and Java 17 for 2022.2+.
-kotlin {
-    jvmToolchain {
-        languageVersion = JavaLanguageVersion.of(17)
-        vendor = JvmVendorSpec.JETBRAINS
-    }
 }
 
-// Configure Gradle IntelliJ Plugin - read more: https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html
+// Configure Gradle IntelliJ Plugin
+// Read more: https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html
 intellij {
-    pluginName = properties("pluginName")
-    version = properties("platformVersion")
-    type = properties("platformType")
-    updateSinceUntilBuild = false
-    // Plugin Dependencies. Uses `platformPlugins` property from the gradle.properties file.
-    plugins = properties("platformPlugins").map { it.split(',').map(String::trim).filter(String::isNotEmpty) }
-}
-
-idea {
-    module {
-        isDownloadJavadoc = true
-        isDownloadSources = true
-    }
-}
-
-configurations {
-    runtimeClasspath {
-        exclude(group = "org.slf4j", module = "slf4j-api")
-    }
-    testImplementation {
-        isCanBeResolved = true
-    }
-}
-
-testlogger {
-    theme = ThemeType.STANDARD
-    showExceptions = true
-    showStackTraces = true
-    showFullStackTraces = false
-    showCauses = true
-    slowThreshold = 2000
-    showSummary = true
-    showSimpleNames = false
-    showPassed = true
-    showSkipped = true
-    showFailed = true
-    showOnlySlow = false
-    showStandardStreams = false
-    showPassedStandardStreams = true
-    showSkippedStandardStreams = true
-    showFailedStandardStreams = true
-    logLevel = LogLevel.LIFECYCLE
-}
-
-// Configure Gradle Changelog Plugin - read more: https://github.com/JetBrains/gradle-changelog-plugin
-changelog {
-    groups.empty()
-    repositoryUrl = properties("pluginRepositoryUrl")
-}
-
-tasks.withType<Test> {
-    environment("GRADLE_RELEASE_REPOSITORY","https://services.gradle.org/distributions")
-    systemProperty("idea.log.leaked.projects.in.tests", "false")
-    systemProperty( "idea.maven.test.mirror", "https://repo1.maven.org/maven2")
-    systemProperty( "com.redhat.devtools.intellij.telemetry.mode", "disabled")
-}
-
-tasks.withType<Copy> {
-    duplicatesStrategy = DuplicatesStrategy.INCLUDE
-}
-
-tasks.register<Test>("integrationTest") {
-    useJUnitPlatform()
-    description = "Runs the integration tests."
-    group = "verification"
-    testClassesDirs = sourceSets["integrationTest"].output.classesDirs
-    classpath = sourceSets["integrationTest"].runtimeClasspath
-    outputs.upToDateWhen { false }
-    mustRunAfter(tasks["test"])
-}
-
-// Configure Gradle Kover Plugin - read more: https://github.com/Kotlin/kotlinx-kover#configuration
-koverReport {
-    defaults {
-        xml {
-            onCheck = true
-        }
-    }
+    version.set("2023.3")
```

Comment:
This bumps the minimum supported version to 2023.3 which should allow us to enable those TextMate-specific behaviors that were incompatible between 2023.2 and 2023.3+.